src/ClockPanel.test.tsx

```diff
@@ -0,0 +1,120 @@
+import { ClockPanel } from "ClockPanel";
+import {render} from '@testing-library/react'
+import '@testing-library/jest-dom'
```

Comment:
Hmmm... you shouldn't need to do this if the [jest-setup](https://github.com/grafana/clock-panel/pull/110/files#diff-1c29b76856438cc695cc32aa47ea0b678d925da633b749ef1b5fd90b882951a0R7) is working correctly.

Reply:
Yep, I should've looked into what jest setup is doing :) 

Removed - https://github.com/grafana/clock-panel/pull/110/commits/2b39f0a664cb97cbff0c16c0d87a3597b7eab641